### PR TITLE
Add person moves tab

### PR DESCRIPTION
**Pull request by thomasleese**
This adds a new tab to the person page for displaying their moves.

### CODE REVIEW

File: app/person/middleware/index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 const setBreadcrumb = require('./set-breadcrumb')
+const setIdentityBar = require('./set-identity-bar')
```
Comment (teneightfive) [inline app/person/middleware/index.js:2]: It's not really a _defined_ pattern but I was trying to find a way to distinguish middleware that sets locals over a property on `req` and I started [prefixing the filenames with `locals`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/main/app/move/app/view/middleware/locals.identity-bar.js).

**Review by teneightfive — APPROVED** [review]
Nice work! I think its a good start to just pull all moves in rather than worrying about splitting them, etc at this stage 👍 

I think one for @ElizabethGoddard to look at when she's back too but keeping this within the 2/3 column width feels a bit squashed:

<kbd>image</kbd>

**Comment by thomasleese** [comment]
> I think one for @ElizabethGoddard to look at when she's back too but keeping this within the 2/3 column width feels a bit squashed:

Yeah, definitely agree. I wondered about changing that, but when I realised it would be the only page without the 2/3 column layout, I thought it was based to wait!

File: app/person/middleware/index.js (hunk 1 above)
Comment (thomasleese) [reply app/person/middleware/index.js]: Good point, I've amended the PR to use this pattern.

File: common/presenters/moves-to-table-component.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            requested: 'govuk-tag--purple',
+            booked: 'govuk-tag--blue',
+            in_transit: 'govuk-tag--yellow',
+            completed: 'govuk-tag--green',
+            cancelled: 'govuk-tag--red',
+            default: 'govuk-tag--blue',
+          }
+
+          const opts = { context: data.status }
+          return componentService.getComponent('govukTag', {
```
[the remark this answers is not in the document]
Comment (thomasleese) [reply common/presenters/moves-to-table-component.js:80]: I'm unsure this is worth refactoring, as the code is so simple.

File: common/presenters/moves-to-table-component.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            text: i18n.t('collections::labels.move_status', opts),
+            classes: classes[opts.context] || classes.default,
+          })
+        },
+      },
+    },
+  ]
+
+  return function buildTable(moves) {
+    return {
```
[the remark this answers is not in the document]
Comment (thomasleese) [reply common/presenters/moves-to-table-component.js:90]: I'm unsure this is worth refactoring, as the code is so simple.

**Comment by thomasleese** [comment]
I'm going to merge this in, the code climate issues feel unnecessary to fix, and we know this page isn't linked to anywhere yet.

**Comment by ElizabethGoddard** [comment]
> > I think one for @ElizabethGoddard to look at when she's back too but keeping this within the 2/3 column width feels a bit squashed:
> 
> Yeah, definitely agree. I wondered about changing that, but when I realised it would be the only page without the 2/3 column layout, I thought it was based to wait!

Hey guys, I think we can afford to go full width on this one like Andy's original designs but @thomasleese I imagine we will chat about this tomorrow in our session :)

**Pull request merged**